content/en/actions/private_actions/update_private_action_runner.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Update the Private Action Runner
+disable_toc: false
+---
+
+## Overview
+
+This page explains how to update the private action runner (PAR). Currently, the PAR is on v{{< private-action-runner-version "private-action-runner" >}}.
```

Comment:
I would add a sentence listing the three different installation methods so users know they need to scroll down to find the appropriate section.